modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -0,0 +1,500 @@
+= snowflake_streaming
+// tag::single-source[]
+:type: output
+:page-beta: true
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+
+Allows Snowflake to ingest data from your data pipeline using https://docs.snowflake.com/en/user-guide/data-load-snowpipe-streaming-overview[Snowpipe Streaming^].
+
+To help you configure your own `snowflake_streaming` output, this page includes two <<example-pipelines,example data pipelines>>.
+
+
+ifndef::env-cloud[]
+Introduced in version 4.39.0.
+endif::[]
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common configuration fields, showing default values
+output:
+  label: ""
+  snowflake_streaming:
+    account: ORG-ACCOUNT # No default (required)
+    user: "" # No default (required)
+    role: ACCOUNTADMIN # No default (required)
+    database: "" # No default (required)
+    schema: "" # No default (required)
+    table: "" # No default (required)
+    private_key: "" # No default (optional)
+    private_key_file: "" # No default (optional)
+    private_key_pass: "" # No default (optional)
+    mapping: "" # No default (optional)
+    init_statement: | # No default (optional)
+      CREATE TABLE IF NOT EXISTS <table-name> (amount NUMBER);
+    schema_evolution:
+      enabled: false # No default (required)
+      new_column_type_mapping: |-
+        root = match this.value.type() {
+          this == "string" => "STRING"
+          this == "bytes" => "BINARY"
+          this == "number" => "DOUBLE"
+          this == "bool" => "BOOLEAN"
+          this == "timestamp" => "TIMESTAMP"
+          _ => "VARIANT"
+        }
```

Comment:
This should be an advanced configuration field.

Reply:
I've removed the field from the common config.